lib/main_common.pm

```diff
@@ -794,6 +795,24 @@ sub map_incidents_to_repo {
     return $ret;
 }
 
+sub join_incidents_to_repo {
+    my ($incidents) = @_;
+    my @repos;
+
+    for my $k (keys %$incidents) {
+        next unless $incidents->{$k};
+        for my $i (split(/,/, $incidents->{k})) {
+            if ($i) {
+                push @repos, $i;
+            }
+        }
+    }
+
+    my $ret = join(',', @repos);
+    $ret =~ s/^,//s;
```

Comment:
With the current code there can't be a comma at the beginning, I would say, so this can be removed and the three lines shortened to
```perl
return join ',', @repos;
```